README.md

```diff
@@ -37,18 +37,14 @@ Table of Contents
 
 # 1. Introduction
 
-This repository is a library for creating th2-data-services applications.
+This repository is an interface for creating th2-data-services applications.
```

Comment:
I think `a library` was better

because we provide not only interfaces but also utils and interface to install source-plugins